### PR TITLE
CR-3512: align tests

### PR DESCRIPTION
**Pull request by Viperain**

### CODE REVIEW

File: cloudrail/knowledge/context/aws/resources_builders/cloudformation/s3_bucket/cloudformation_s3_bucket_builder.py [hunk 1]
```diff
@@ -62,3 +63,31 @@ def parse_resource(self, cfn_res_attr: dict) -> Optional[S3BucketLogging]:
                                    cfn_res_attr['region'])
         else:
             return None
+
+class CloudformationS3BucketAclBuilder(CloudformationS3BucketBuilder):
```
Comment (Imanuel-Mizrahi) [inline cloudrail/knowledge/context/aws/resources_builders/cloudformation/s3_bucket/cloudformation_s3_bucket_builder.py:67]: @idan-aviv , please review this builder

File: cloudrail/knowledge/context/aws/resources_builders/cloudformation/s3_bucket/cloudformation_s3_bucket_builder.py [hunk 2]
```diff
@@ -62,3 +63,31 @@ def parse_resource(self, cfn_res_attr: dict) -> Optional[S3BucketLogging]:
...
+                return [S3ACL(S3Permission['FULL_CONTROL'], GranteeTypes.CANONICAL_USER, None, bucket_name, account, region),
+                        S3ACL(S3Permission['READ'], GranteeTypes.GROUP, S3PredefinedGroups.ALL_USERS.value, bucket_name, account, region),
+                        S3ACL(S3Permission['WRITE'], GranteeTypes.GROUP, S3PredefinedGroups.ALL_USERS.value, bucket_name, account, region)]
+            if canned_acl == 'AuthenticatedRead':
+                return [S3ACL(S3Permission['FULL_CONTROL'], GranteeTypes.CANONICAL_USER, None, bucket_name, account, region),
+                        S3ACL(S3Permission['READ'], GranteeTypes.GROUP, S3PredefinedGroups.AUTHENTICATED_USERS.value, bucket_name, account, region)]
+            if canned_acl == 'LogDeliveryWrite':
+                return [S3ACL(S3Permission['WRITE'], GranteeTypes.GROUP, S3PredefinedGroups.LOG_DELIVERY.value, bucket_name, account, region),
+                        S3ACL(S3Permission['READ_ACP'], GranteeTypes.GROUP, S3PredefinedGroups.LOG_DELIVERY.value, bucket_name, account, region)]
+            if canned_acl == 'AwsExecRead':
```
Comment (idan-aviv) [inline cloudrail/knowledge/context/aws/resources_builders/cloudformation/s3_bucket/cloudformation_s3_bucket_builder.py:90]: @Imanuel-Mizrahi giving full access and than read (contained) access for the same grantee...can delete the read permission

File: cloudrail/knowledge/context/aws/resources_builders/cloudformation/ec2/cloudformation_security_group_builder.py [hunk 3]
```diff
@@ -13,11 +13,16 @@ def __init__(self, cfn_by_type_map: Dict[CloudformationResourceType, Dict[str, D
 
     def parse_resource(self, cfn_res_attr: dict) -> SecurityGroup:
         properties: dict = cfn_res_attr['Properties']
-        security_group: SecurityGroup = SecurityGroup(security_group_id=self.get_resource_id(cfn_res_attr),
```
Comment (idan-aviv) [inline cloudrail/knowledge/context/aws/resources_builders/cloudformation/ec2/cloudformation_security_group_builder.py:16]: @Imanuel-Mizrahi redundant false as default value
and shorter expression:
security_group_id  = vpc_id or self.get_resource_id(cfn_res_attr)

**Review by Imanuel-Mizrahi — APPROVED** [review]

File: cloudrail/knowledge/context/aws/resources_builders/cloudformation/ec2/cloudformation_security_group_builder.py [hunk 4]
```diff
@@ -13,11 +13,16 @@ def __init__(self, cfn_by_type_map: Dict[CloudformationResourceType, Dict[str, D
...
+        vpc_id = self.get_property(properties, 'VpcId', False)
+        security_group_id = self.get_resource_id(cfn_res_attr) if vpc_id else None
```
Comment (idan-aviv) [inline cloudrail/knowledge/context/aws/resources_builders/cloudformation/ec2/cloudformation_security_group_builder.py:17]: @Imanuel-Mizrahi GroupName isn't require field so why enforcing pseudo value?
 this is enough:
name = self.get_property(properties, 'GroupName', self.get_name_tag(properties))

**Pull request merged**